src/pom.xml

```diff
@@ -1022,9 +1022,9 @@
         <version>${slf4j.version}</version>
       </dependency>
       <dependency>
-        <groupId>log4j</groupId>
-        <artifactId>log4j</artifactId>
-        <version>1.2.17.norce</version>
+        <groupId>ch.qos.reload4j</groupId>
+        <artifactId>reload4j</artifactId>
+        <version>1.2.19</version>
```

Comment:
Perhaps use a property here (like the `slf4j.version` approach above)?

Reply:
I can do so, I do not plan to keep reload4j long (just using it to prove geotools and geowebcache changes are effective)